frontend/src/metabase-lib/v1/queries/utils/segments.js

```diff
@@ -9,21 +9,8 @@ export function getSegmentOrMetricQuestion(query, table, metadata) {
 export function getDefaultSegmentOrMetricQuestion(table, metadata) {
   const question = metadata.table(table.id).question();
 
-  if (table.entity_type === "entity/GoogleAnalyticsTable") {
-    const dateField = table.fields.find(f => f.name === "ga:date");
-    if (dateField) {
-      return question
-        .legacyQuery({ useStructuredQuery: true })
-        .filter(["time-interval", ["field", dateField.id, null], -365, "day"])
-        .aggregate(["metric", "ga:users"])
-        .question();
-    }
-  } else {
-    return question
-      .legacyQuery({ useStructuredQuery: true })
-      .aggregate(["count"])
-      .question();
-  }
-
-  return null;
```

Comment:
🤔

Reply:
Just puzzled why we were ever returning null here...

Reply:
Oooooooor

if entity type was `GoogleAnalyticsTable`, but NOT `dateField` => return `null`?

Weird placement, nevertheless.

Reply:
Right, I missed that